metrics/grafana/tidb_runtime.json

```diff
@@ -1517,6 +1517,314 @@
             "align": false,
             "alignLevel": null
           }
+        },
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "${DS_TEST-CLUSTER}",
+          "description": "",
+          "fieldConfig": {
+            "defaults": {},
+            "overrides": []
+          },
+          "fill": 1,
+          "fillGradient": 0,
+          "gridPos": {
+            "h": 7,
+            "w": 12,
+            "x": 0,
+            "y": 50
+          },
+          "hiddenSeries": false,
+          "id": 31,
+          "legend": {
+            "avg": false,
+            "current": false,
+            "max": false,
+            "min": false,
+            "show": true,
+            "total": false,
+            "values": false
+          },
+          "lines": true,
+          "linewidth": 1,
+          "nullPointMode": "null",
+          "options": {
+            "alertThreshold": true
+          },
+          "percentage": false,
+          "pluginVersion": "7.5.11",
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "exemplar": true,
+              "expr": "time()-tidb_disttask_ddl_subtask_start_time{k8s_cluster=\"$k8s_cluster\",tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", status=\"pending\"}",
```

Comment:
why not put this in previous `Dist execute frameowork`?

Reply:
we can see this detail in each TiDB Node. If we put it in Dist execute frameowork, different tidb details are mixed.

Reply:
you need `k8s_cluster="$k8s_cluster", tidb_cluster="$tidb_cluster", instance=~"$instance"` labels

Reply:
I didn't find other metrics containing these lables 🤔

Reply:
In this way, we can see the details of each TiDB, rather than being mixed in one panel.
![image](https://github.com/pingcap/tidb/assets/23556834/73c9d89d-3797-4483-b360-18da90361083)